### ISSUE TITLE
When the member names are the same configuration is ignored. 

### ISSUE THREAD

**Issue by rmbrunet** [fluentsprings/ExpressMapper#99]
Priviet Yuriy!

I noticed that when the member names are the same in the source and destination classes then configuration is ignored. My code looks like:

```
Mapper.Register<Source, Target>().Function(target => target.Enabled, source => source.enabled == "Y")
```

where Source.Enabled is an string while Target.Enabled is a bool. And the mapping fails with FormatException "String was not recognized as a valid Boolean".

Great Library! Thanks for your work.

Roberto.

**Comment by anisimovyuriy** [comment]
Hola Roberto!

Thanks a lot for your valuable input and contribution! Let me check and I will get back to you soon. Just a hint for you: In your case I'd use ".Member" instead of ".Function" - it is faster.

Thanks,
Yuriy

**Comment by rmbrunet** [comment]
Got it. Thanks! 

This is a sample of the issue with a Linqpad script:

```
void Main()
{
    Mapper.Reset();
    Mapper.Register<Source, Target>().Member(t => t.Enabled, s => s.enabled == "Y");
    Mapper.Compile();

    var source = new Source { enabled = "N" };

    try
    {
        Mapper.Map<Source, Target>(source).Dump();
    }
    catch (FormatException ex) {
        ex.Message.Dump();
    }
    
}

class Source
{
    public string enabled {get; set;}
}

class Target
{
    public bool Enabled { get; set;}
}
```

**Comment by rolandoldengarm** [comment]
I am facing the same issue. I'm using .Member to HTML encode my values from the source object. That is just being ignored...
Any idea what the ETA of the fix is?
Or is there a workaround, except of course renaming the target properties?

**Comment by anisimovyuriy** [comment]
Fixed! Will be included in 1.8.5 release.
Thanks @rolandoldengarm and @rmbrunet for your valuable input and contribution!
